contracts/token/CRC1155/extensions/ICRC1155Metadata.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC1155/extensions/IERC1155MetadataURI.sol";
+
+/**
+ * @dev Optional metadata extension.
+ */
+interface ICRC1155Metadata is IERC1155MetadataURI {
```

Comment:
Why is this `IERC1155MetadataURI`?

Reply:
`IERC1155MetadataURI` comes from OZ, and the new `ICRC1155Metadata` is used for extension. This is helpful when CRC1155 is used as a pure NFT contract.